Src/FluentAssertions/TypeExtensions.cs

```diff
@@ -68,5 +68,71 @@ public static PropertyInfoSelector Properties(this TypeSelector typeSelector)
         {
             return new PropertyInfoSelector(typeSelector.ToList());
         }
+
+        public static bool IsEnumerable(this Type type)
```

Comment:
Keep all these methods `internal` please.

Reply:
All the other methods in this class are `public`...

Reply:
Yeah, there's some confusion _and_ technical debt. The `TypeExtensions` in the root of the project is part of the public API of the type assertions. There's another one under `Common`, which is supposed to be used within the framework and _should_ have been ìnternal`.